test/unit/renderer/viz/expressions/Scale.test.js

```diff
@@ -0,0 +1,10 @@
+import { validateStaticType } from './utils';
+
+describe('src/renderer/viz/expressions/zoom', () => {
```

Comment:
Some tests would be great ✨ and use `src/renderer/viz/expressions/scale` instead of `src/renderer/viz/expressions/zoom`

Reply:
Yes, I'll fix the typo, regarding tests, this is lacking render tests actually (this cannot be tested with unit tests without coupling to the rendering implementation)